src/__test__/index.test.ts

```diff
@@ -13,9 +13,34 @@ test('Mock functions', async () => {
     // Specify the returned data of the mockedAxios once
     mockedAxios.get.mockImplementationOnce(() => Promise.resolve(resp));
     // TODO: Use mock function to get response object
-    // const enforcer = new Enforcer('http://localhost:4000');
-    // enforcer.setUser('alice');
-    // expect(enforcer.getProfiles()).toMatchObject(respObj);
+    // const authorizer = new Authorizer('http://localhost:4000');
+    // authorizer.setUser('alice');
+    // expect(authorizer.getProfiles()).toMatchObject(respObj);
 
     // expect(mockedAxios.get('http://localhost:4000/api/casbin?casbin_subject=alice')).toMatchObject(respObj);
 });
+
+test('Authorizer', () => {
+    const profilesObj = {
+        read: ['data1', 'data2', 'data3'],
+        write: ['data1']
+    }
+    const authorizer = new Authorizer();
+    authorizer.setProfiles(JSON.stringify(profilesObj));
```

Comment:
I prefer using `authorizer.setProfiles(profilesObj);`

Reply:
Actually `setProfiles` (now it is renamed as `setPermission`) accept a string in JSON format as the input. The `profilesObj` here is a JS object, so I stringify it before passing it to the function.